components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FormattingUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.gradle.enterprise.cli;
+
+import java.time.Duration;
+import java.util.Locale;
+
+final class FormattingUtils {
+
+    private FormattingUtils() {
+    }
+
+    static String formatDuration(Duration duration) {
+        long hours = duration.abs().toHours();
+        long minutes = duration.abs().toMinutes() % 60;
+        double seconds = (duration.abs().toMillis() % 60_000) / 1000d;
+
+        StringBuilder s = new StringBuilder();
+        if (duration.isNegative()) {
+            s.append('-');
+        }
+        if (hours != 0) {
+            s.append(hours).append("h ");
+        }
+        if (minutes != 0) {
+            s.append(minutes).append("m ");
+        }
+        s.append(String.format(Locale.ROOT, "%.3fs", seconds));
+
+        return s.toString().trim();
+    }
```

Comment:
I would remove the blank line above the return statement.

Reply:
Done in 5349d037a38ecf0fa07b9b1884575d4d807e9c46